src/MagnumPlugins/MeshOptimizerSceneConverter/Test/MeshOptimizerSceneConverterTest.cpp

```diff
@@ -1131,6 +1144,169 @@ void MeshOptimizerSceneConverterTest::simplifyVerbose() {
 )";
     CORRADE_COMPARE(out.str(), expected);
 }
+void MeshOptimizerSceneConverterTest::encodeDecodeInterleavedMesh() {
+    Containers::Pointer<AbstractSceneConverter> converter = _manager.instantiate("MeshOptimizerSceneConverter");
+    converter->configuration().setValue("encodeVertex", true);
+
+    struct QuadVertex {
```

Comment:
What makes this vertex a "Quad" rather than just a vertex? Possibly just `Vertex` is sufficient :)